README.md

```diff
@@ -523,50 +558,71 @@ Directory in which to store backups.
 
 ##### `backupdirmode`
 
-Permissions applied to the backup directory. This parameter is passed directly
-to the `file` resource.
+Permissions applied to the backup directory. This parameter is passed directly to the `file` resource.
 
 ##### `backupdirowner`
 
-Owner for the backup directory. This parameter is passed directly to the `file`
-resource.
+Owner for the backup directory. This parameter is passed directly to the `file` resource.
 
 ##### `backupdirgroup`
 
-Group owner for the backup directory. This parameter is passed directly to the
-`file` resource.
+Group owner for the backup directory. This parameter is passed directly to the `file` resource.
 
 ##### `backupcompress`
 
-Whether backups should be compressed. Valid values are true, false. Defaults to true.
+Whether backups should be compressed.
+
+Valid values are true, false.
+
+Defaults to true.
 
 ##### `backuprotate`
 
-How many days to keep backups. Valid value is an integer. Defaults to '30'.
+How many days to keep backups.
+
+Valid value is an integer.
+
+Defaults to '30'.
```

Comment:
This 30 is a numeric, so actually shouldn't have single quotes. I know you didn't add them, but please make them go away?